services/src/modules/opa-helper.spec.ts

```diff
@@ -1,10 +1,10 @@
-import { getTmpFilePaths, prepareCompiledRegoFile, normalizeRegoCodePackage } from './opaHelper';
-import { mocked } from 'ts-jest/utils';
 import { promises as fs } from 'fs';
 import { exec } from 'child_process';
-import { tmpPoliciesDir } from './config';
 import * as path from 'path';
-import mockFsForOpa from '../tests/helpers/mockFsForOpa';
+import { mocked } from 'ts-jest/utils';
+import * as mockFsForOpa from '../tests/helpers/mock-fs-for-opa';
```

Comment:
It isn't default export. The mock-fs-for-opa hasn't default export. It exports 2 functions

Reply:
It has a default export that exposes the 2 functions: `export default { mock, restore }`, and it was being used before

This is so you can import it like it was before (`import mockFsForOpa from '../tests/helpers/mockFsForOpa'`) instead of having to use `* as` like after the change.
I still don't see a good reason not to use the default